### PR TITLE
Add flags to prtagbuilder

### PR DESCRIPTION
**Pull request by dekiel**
Added support for finding PR number for branch HEAD provided as cli flags.
With flag -O prtagbuilder print pr number only.

### CODE REVIEW

**Review by Ressetkk — DISMISSED** [review]
I have left some additional insights. I think it's best not to exit program on package level. This will block us if we wanted to re-use this package. The function should return an error and then we should handle errors outside.

**Review by tehcyx — DISMISSED** [review]

File: development/tools/cmd/prtagbuilder/main.go [hunk 1]
```diff
@@ -1,18 +1,73 @@
...
+					}
+				}
+			} else if cmd.Flags().NFlag() > 2 {
+				err := checkFlags(cmd.Flags())
+				if err != nil {
+					logrus.WithError(err).Fatalf("Required flag is empty")
+				}
+				// set prtagbuilder to use data from flags
+				fromFlags = true
+			}
```
Comment (tehcyx) [inline development/tools/cmd/prtagbuilder/main.go:35]: Suggesting to rewrite for readability:
```go
numFlags := cmd.Flags().NFlag()
if numFlags == 1 && !cmd.Flags().Changed("numberOnly") || numFlags > 2 {
    err := checkFlags(cmd.Flags())
    if err != nil {
        logrus.WithError(err).Fatalf("Required flag is empty")
    }
    if numFlags > 2 {
        fromFlags = true
    }
}
```

Comment (tehcyx) [reply development/tools/cmd/prtagbuilder/main.go:35]: The last if statement can also be removed if you do it like this:
```go
fromFlags = numFlags > 2
```

**Review by tehcyx — APPROVED** [review]

**Review by tehcyx — APPROVED** [review]
Looks good now

**Pull request merged**